RELEASE.md

```diff
@@ -37,6 +37,7 @@
 ## Documentation changes
 * Improved documentation for configuring dataset parameters in the data catalog
 * Extended documentation with an example of logging customisation at runtime
+* Added docstrings in three private functions in `kedro/utils.py`
```

Comment:
```suggestion
```

We don't add changes like these into the release notes, because they're not technically user facing changes. Especially because these docstrings have been added to private methods.

Reply:
Agree, thanks @merelcht !